### PR TITLE
Gem sivapuja

### PR DESCRIPTION
**Pull request by SivaPujaPasupulati**
1. Verified Gherkin syntax and usage wrt Given, Then, When, And functionalities updated Feature file in the same way
2. Updated CSS xpath with normal xpath 
3. Updated boolean variable with proper name

### CODE REVIEW

**Comment by SivaPujaPasupulati** [comment]
Hi @gem-akatiha Test Control 15 Automated scenarios are pushed.

File: src/main/java/com/qa/gemini/locators/TestControlLocators.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    public static String calendarSubmit = "//span[text()='Submit']";
+
+    public static String totalRecords = "//span[contains(text(), 'Total Records')]";
+
+    public static String examTemplate = "//span[text()='Exam Template Management']";
+
+    public static String exportButton = "//button[@label='Export']";
+
+    public static String recordsData = "//button[contains(@class,'p-paginator-page p-paginator-element p-link p-highlight p-ripple ng-star-inserted')]";
```
Comment (gem-akatiha) [inline src/main/java/com/qa/gemini/locators/TestControlLocators.java:26]: There is no need to include all the classes if using contains

File: src/main/java/com/qa/gemini/stepdefinitions/CandidateFlow.java [hunk 2]
```diff
@@ -0,0 +1,1278 @@
...
+        }catch(Exception e){
+            GemTestReporter.addReasonOfFailure(e+" Exception occurred while verifying the default tab selected.");
+        }
+    }
+
+
+    @Then("switch to {string}")
+    public void switchTo(String tab) {
+        try {
+            if(DriverAction.isExist(By.xpath(CandidateFlowLocators.testTab.replace("input", tab)),5000));
```
Comment (gem-akatiha) [inline src/main/java/com/qa/gemini/stepdefinitions/CandidateFlow.java:390]: if statement is without any block

File: src/main/java/com/qa/gemini/stepdefinitions/TestControlScreen.java [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+        GemTestReporter.addTestStep("Verify if records are filtered based on count","Count of Filtered records based on a parameter are: "+totalRecords, STATUS.PASS);
+    }
+
+    @When("enter search text in Search by name or tag")
+    public void enterSearchTextInSearchByNameTag() {
+        waitUntilElementAppear(By.xpath("//input[@type='search']"),10);
+        waitSec(5);
+        typeText(By.xpath("//input[@type='search']"),"Java");
+        waitSec(5);
+        GemTestReporter.addTestStep("Verify entered text","Entered text is: Java",STATUS.PASS);
```
Comment (gem-akatiha) [inline src/main/java/com/qa/gemini/stepdefinitions/TestControlScreen.java:158]: why do we have declared xpaths in the scripts directly?

**Review by gem-akatiha — APPROVED** [review]

**Pull request merged**